src/styles/print.scss

```diff
@@ -0,0 +1,57 @@
+// This file contains styles for printed pages.
+// stylelint-disable declaration-no-important, selector-no-qualifying-type
+@import "app-styling-utils";
+@import "app-styling-variables";
+
+
+@media print {
+  html,
+  body {
+    background-color: transparent !important;
+    color: app-get-color($app-text-colors, "normal") !important;
+    font-family: "Times New Roman", serif;
```

Comment:
I would like to include a nice font in the project.

Reply:
Will get to that when we start creating pages. They will look very ugly at the beginning.